docs/docs/guide/introduction/cost-of-convenience.md

```diff
@@ -5,7 +5,7 @@ description: Learn about the cost of convenience in Xcode and how Tuist helps yo
 
 # The cost of convenience
 
-Before continuing, let's take a moment to consider the cost of convenience in [Xcode](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://developer.apple.com/xcodenp/&ved=2ahUKEwik55Xo35-FAxVqxzgGHZAZCz4QFnoECAYQAQ&usg=AOvVaw28WTg1BlCGAf937pcwXMip).
```

Comment:
- not sure why we need the whole redirect via google.com here 🤷 
- destination https://developer.apple.com/xcodenp leads to Page Not Found 🔍